main.go

```diff
@@ -1227,6 +1227,19 @@ func (git *repoSync) removeStaleWorktrees() (int, error) {
 	return count, nil
 }
 
+func hasGitLockFile(gitRoot absPath) (bool, error) {
```

Comment:
How about also returning the name of the first detected lockfile which , so the caller can log it?